modules/sdk-coin-apt/src/lib/utils.ts

```diff
@@ -1,11 +1,13 @@
 import {
```

Comment:
check the warnings in the file.

<img width="452" alt="image" src="https://github.com/user-attachments/assets/ad9acfc6-cad1-4627-a0ba-0d93f4ce6e5e" />